cypher-shell/src/main/java/org/neo4j/shell/cli/InteractiveShellRunner.java

```diff
@@ -168,13 +170,23 @@ AnsiFormattedText getPrompt() {
         //  but that does not work in general)
         databaseName = ABSENT_DB_NAME.equals(databaseName) ? DEFAULT_DEFAULT_DB_NAME : databaseName;
 
-        AnsiFormattedText prompt = AnsiFormattedText.s().bold()
+        int promptIndent = connectionConfig.username().length() + 1 + databaseName.length() + 2;
```

Comment:
Can you add a comment what the +1 and +2 is for?

Reply:
Ya, made constants.